recipe/meta.yaml

```diff
@@ -1,37 +1,38 @@
 {% set name = "omegaconf" %}
-{% set version = "1.4.1" %}
+{% set version = "2.0.0rc26" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/omry/{{ name|lower }}/archive/{{ version }}.tar.gz
-  sha256: f9078df667ed61ec4852c3d20bf7ffa0fbfc3c941f4a95dc71f4ce96d0494ccd
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b62ad9288272c601a5a6a31f7de0efbc26568ec6a42ecd485b0320fd913b8953
 
 build:
   noarch: python
-  number: 2
+  number: 0
   script: "{{ PYTHON }} -m pip install . -vv"
 
 requirements:
   host:
-    - python
+    - python >=3.6
     - pip
     - pytest-runner
   run:
-    - python
+    - python >=3.6
     - pyyaml
-    - six
+    - dataclasses
```

Comment:
is it possible to condition the dataclass dependency on the python version being 3.6 ?

Reply:
I removed it since it's the only way to get a norach conda package for omegaconf. The dataclasses conda package is empty for python >3.6: https://github.com/conda-forge/dataclasses-feedstock/pull/3

Reply:
It's kind of hacky but I don't think we have another choice here. Maybe a conda-forge dev can comment on this?

Reply:
This is okay for now but I would like to know that there is a plan to address this. the dataclasses package is bringing dataclasses to Python 3.6 and is not required in newer versions.

In pip land this is expressed as 
```
dataclasses;python_version=='3.6'
```

I would be good if Conda supported something similar.

Reply:
The alternative is to make this package not `noarch` but I don't know whether it's preferable or not on a conda-forge point of view. Ping @beckermr @CJ-Wright

Reply:
noarch packages use less of our CI resources - that being said we are doing ok on them, so i'd say if you have a real need, then by all mean make the package non-noarch.

Reply:
@beckermr I have a lint error

`Non noarch packages should have python requirement without any version constraints.`

This package is also only for python >= 3.6.

Reply:
I vote to make it an arch specific package then. what would that look like in the recipe?
can we special case Python 3.6 and have newer Python versions use the same definition?